utils/after_success.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# This script is meant to be used by Travis-CI to publish artifacts (binary, sorce and javadoc jars) when releasing snapshots.
+# Right now it will be manually triggered to push jars to nexus server.
+
+SITE_VERSION=$(./gradlew properties | grep -o 'version.*' | sed -n -e 's/^.*version: //p')
+if [ "$SITE_VERSION" == "" ]; then
+    echo "Could not determine the version, so we're exiting."
+    exit 1
+fi
+if [ "${SITE_VERSION##*-}" != "SNAPSHOT" ]; then
+    URL=http://104.197.230.53:8081/nexus/content/repositories/snapshots/io/gapi/gapi-tools-framework/$SITE_VERSION/
```

Comment:
I'm not sure what you mean there...do you not have access to that file when running these scripts?